### PR TITLE
changed content in interview requirements section

### PR DESCRIPTION
**Pull request by EmmaFrith**
replaced interview preferences with requirements and amended content on interview page followiung university feedback

### CODE REVIEW

File: app/data/session-data-defaults.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
     self_amend: process.env.MVP !== 'true',
     self_amend_contact_details: process.env.MVP !== 'true',
     self_withdraw: process.env.MVP !== 'true',
-    interview_preferences: false
+    interview_preferences: true
```
Comment (adamsilver) [inline app/data/session-data-defaults.js]: @paulrobertlloyd is this like a feature toggle?

File: app/routes/application/review.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ module.exports = router => {
           english: 'English GCSE or equivalent not entered',
           science: 'Science GCSE or equivalent not entered'
         },
-        interview: 'Tell us your interview preferences',
+        interview: 'Tell us your Interview requirements',
```
Comment (adamsilver) [inline app/routes/application/review.js]: Lowercase ‘I’

File: app/views/_includes/review/interview.njk [hunk 3]
```diff
@@ -13,7 +13,7 @@
         items: [{
           href: applicationPath + "/interview?referrer=" + referrer,
           text: "Change" if applicationValue(["interview"]) else "Add",
-          visuallyHiddenText: "interview preferences"
+          visuallyHiddenText: "Interview requirements"
```
Comment (adamsilver) [inline app/views/_includes/review/interview.njk]: Lowercase “I”

File: app/views/_includes/review/interview.njk [hunk 4]
```diff
@@ -29,7 +29,7 @@
 {% else %}
   {% set incompleteType = "warning" if errorList %}
   {% set incompleteId = "interview" %}
-  {% set incompleteText = "Tell us your interview preferences" %}
+  {% set incompleteText = "Tell us your Interview requirements" %}
```
Comment (adamsilver) [inline app/views/_includes/review/interview.njk]: Lowercase “I”

File: app/data/session-data-defaults.js (hunk 1 above)
Comment (paulrobertlloyd) [reply app/data/session-data-defaults.js]: It is! Can access within the prototype at `data.flags.interview_preferences`

File: app/views/application/interview/index.njk [hunk 5]
```diff
@@ -18,17 +18,26 @@
...
+
+  <p class="govuk-body">For example:</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>any disability, health or personal issue you feel is relevant</li>
-    <li>special arrangements or assistance you may need</li>
-    <li>dates you are unavailable for interview</li>
+    <li>you have commitments such as employment or caring responsibilities</li>
+    <li>you'll be travelling a long way to get to the interview</li>
+    <li>you'll need some adjustments because you're disabled</li>
+    &nbsp
```
Comment (adamsilver) [inline app/views/application/interview/index.njk]: This should be removed and the `<ul>` put back where it was. Let's chat through how to add space. :)

**Pull request merged**